<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BossIntro</class>
 <widget class="QDialog" name="BossIntro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BossPlayButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QLabel" name="intro_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>271</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>按W、A、S、D移动</string>
   </property>
  </widget>
  <widget class="QLabel" name="intro_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>按J发射子弹</string>
   </property>
  </widget>
  <widget class="QLabel" name="intro_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>按Q键冲刺</string>
   </property>
  </widget>
  <widget class="QLabel" name="intro_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>按Esc键暂停</string>
   </property>
  </widget>
  <widget class="QLabel" name="intro_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>271</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>战狠，昊京，八龙孙国帅捅了老挝。</string>
   </property>
  </widget>
  <widget class="QLabel" name="intro_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>271</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>请你把他击败，不要被他绳之以法。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
